digdag-tests/src/test/java/acceptance/AccessControllerIT.java

```diff
@@ -138,6 +143,14 @@ public void bindAuthorization()
                         {
                             binder().bind(AccessController.class).to(ProjectBasedAccessController.class);
                         }
+
+                        @Override
+                        protected void bindAuthenticator()
+                        {
+                            binder().bind(JwtAuthenticatorConfig.class).toProvider(JwtAuthenticatorConfigProvider.class).asEagerSingleton();
+                            binder().bind(Authenticator.class).to(JwtAuthenticator.class);
+                        }
+
```

Comment:
It is necessary to overwrite `bindAuthenticator` by changing the specifications of` ServerBootstrap`.
Originally, `v0.10` uses` JwtAuthenticator` as `Authenticator`, so it binds JwtAuthenticatorConfig etc.